go.mod

```diff
@@ -12,13 +12,8 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.2.0
 	github.com/Azure/go-autorest/autorest/to v0.3.0
 	github.com/Azure/go-autorest/autorest/validation v0.2.0 // indirect
-<<<<<<< HEAD
-	github.com/crossplaneio/crossplane v0.6.0-rc.0.20191226165033-a452562456e0
-	github.com/crossplaneio/crossplane-runtime v0.3.0
-=======
```

Comment:
Looks like some merge conflicts need resolving.

Reply:
Oh hmm thought I got those. One moment.

Reply:
@negz oh I see I think you were looking at a previous revision